pom.xml

```diff
@@ -1532,6 +1549,31 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
           </exclusion>
+          <!-- Hive 2.3.4 -->
+          <exclusion>
+            <groupId>${hive.group}</groupId>
+            <artifactId>hive-vector-code-gen</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>${hive.group}</groupId>
+          <artifactId>hive-llap-tez</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.apache.orc</groupId>
+            <artifactId>orc-tools</artifactId>
+          </exclusion>
+          <exclusion>
```

Comment:
Exclude `avatica-metrics` and `avatica`.  more details: https://issues.apache.org/jira/browse/SPARK-27054